e2e/ruby/rails/demo/Gemfile

```diff
@@ -6,7 +6,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '~> 3.3.4'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem 'rails', '~> 7.0.8', '>= 7.0.8.1'
+gem 'rails', '~> 7.1.0'
```

Comment:
Had to upgrade to use `broadcast_to`, which I thought might be part of the issue I was troubleshooting.